content/3_Silver/Graph_Traversal.mdx

```diff
@@ -1082,6 +1082,49 @@ public class BuildingTeams {
 ```
 
 </JavaSection>
+<PySection>
+
+<Warning>
+You have to submit with CPython (not PyPy3) to avoid exceeding the time limit.
+</Warning>
+
+```py
+import sys
+
+input = sys.stdin.readline
+
+sys.setrecursionlimit(int(1e5 + 1))
```

Comment:
magic #

Reply:
For some reason, n + 3 is the magic number

Reply:
would you mind setting it to 10 ** 9
like add a comment "disable recursion limit"